artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -3272,7 +3273,8 @@ public Queue createQueue(final AddressInfo addrInfo,
                             final long autoDeleteDelay,
                             final long autoDeleteMessageCount,
                             final boolean autoCreateAddress,
-                            final boolean configurationManaged) throws Exception {
+                            final boolean configurationManaged,
+                            final long ringSize) throws Exception {
```

Comment:
breaking api change, new param needs a new method added (old method interface honoured)

Reply:
Hmmm...Even though this is public it isn't part of the interface so I'm inclined to leave this.